app/src/main/res/layout/fragment_browser.xml

```diff
@@ -14,6 +14,52 @@
         android:layout_height="match_parent"
         android:orientation="vertical">
 
+        <LinearLayout
+            android:id="@+id/bannerLayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:background="#FEBD4F"
+            android:gravity="center">
+
+          <TextView
```

Comment:
Ah, a few localization concerns:
- should we specify a max line count (or height) for this view? In a localized language, the text may overflow the banner, in the worst case covering the entire screen
- Similarly, in localized languages the dismiss and more info buttons could grow very large horizontally, further pushing this content down (or possibly hiding it entirely). Should we specify a max width for these buttons?